tests/components/scorecard_chart.test.ts

```diff
@@ -128,28 +116,284 @@ describe("Scorecard charts", () => {
     await nextTick();
     expect(getChartElement()).toMatchSnapshot();
   });
+});
+
+describe("Scorecard charts rendering", () => {
+  beforeEach(async () => {
+    chartId = "someuuid";
+    sheetId = "Sheet1";
+    const data = {
+      sheets: [
+        {
+          name: sheetId,
+          colNumber: 10,
+          rowNumber: 10,
+          rows: {},
+          cells: {
+            A1: { content: "2" },
+            A2: { content: "3" },
+            A3: { content: "2.1234" },
+            B1: { content: "1" },
+            B2: { content: "2" },
+            B3: { content: "3" },
+          },
+        },
+      ],
+    };
+    ({ model, fixture } = await mountSpreadsheet({ model: new Model(data) }));
```

Comment:
It's probably now possible to avoid having to mount a spreadsheet and waiting for nextTicks in the test, just testing directly the scorecard with a random canvas ?

That'd make the tests faster :D